include/date.h

```diff
@@ -0,0 +1,8234 @@
+#ifndef DATE_H
+#define DATE_H
```

Comment:
Cool lib - but please add it either as a git submodule (available for both emulator and hardware) or as a dependency into platformio (available for hardware).